tests/sharedtria/dof_01.cc

```diff
@@ -92,19 +92,20 @@ void test()
       triangulation.execute_coarsening_and_refinement ();
       dof_handler.distribute_dofs (fe);
 
+      // avoid outputing any partitioning info until Parmetis gives reproducable results
       deallog
-          << "n_dofs: " << dof_handler.n_dofs() << std::endl
-          << "n_locally_owned_dofs: " << dof_handler.n_locally_owned_dofs() << std::endl;
-
-      deallog << "n_locally_owned_dofs_per_processor: ";
-      std::vector<types::global_dof_index> v = dof_handler.n_locally_owned_dofs_per_processor();
-      unsigned int sum = 0;
-      for (unsigned int i=0; i<v.size(); ++i)
-        {
-          deallog << v[i] << " ";
-          sum += v[i];
-        }
-      deallog << " sum: " << sum << std::endl;
+          << "n_dofs: " << dof_handler.n_dofs() << std::endl;
+//          << "n_locally_owned_dofs: " << dof_handler.n_locally_owned_dofs() << std::endl;
+
+//      deallog << "n_locally_owned_dofs_per_processor: ";
+//      std::vector<types::global_dof_index> v = dof_handler.n_locally_owned_dofs_per_processor();
+//      unsigned int sum = 0;
```

Comment:
Ah, I didn't see this -- thanks for pointing it out. In that case I'm ok with it.

Do you think it's worthwhile keeping this text commented out? Would you ever want to bring them back in? If not, take the commented lines out and merge yourself. If you think we may eventually want to bring them back in (e.g., if we replace the partitioning code by something homegrown) then just merge as is.